redis_sentinel/datadog_checks/redis_sentinel/redis_sentinel.py

```diff
@@ -130,6 +139,9 @@ def _process_slaves_stats(self, redis_conn, master_name, base_tags, master_tags)
                 tags=slave_tags,
             )
 
+        self.gauge('redis.sentinel.odown_slaves', slaves_odown, tags=master_tags)
+        self.gauge('redis.sentinel.sdown_slaves', slaves_sdown, tags=master_tags)
```

Comment:
Might be good to have some tests for `redis.sentinel.odown_slaves` and `redis.sentinel.sdown_slaves`  to assert the `slaves_odown` and `slaves_sdown` counters.